vignettes/r2dii-analysis.Rmd

```diff
@@ -102,28 +89,25 @@ sda_targets <- matched %>%
     co2_intensity_scenario = co2_intensity_scenario_demo
   )
 
-sda_targets %>% head()
+sda_targets
 ```
 
 ## Visualization
 
-There are a large variety of possible visualizations stemming from the outputs 
-of `target_market_share()` and `target_sda()`. Below, we have highlighted a couple of 
-common plots, along with the minimum code necessary to reproduce them. 
+There are a large variety of possible visualizations stemming from the outputs of `target_market_share()` and `target_sda()`. Below, we have highlighted a couple of common plots, along with the minimum code necessary to reproduce them. 
 
 ### Market Share: Sector-level technology mix
 
-From the market share output, you can plot the portfolio's exposure to various 
-climate sensitive technologies (`projected`), and compare with various scenario 
-targets.
+From the market share output, you can plot the portfolio's exposure to various climate sensitive technologies (`projected`), and compare with various scenario targets.
 
 ```{r technoloy mix portfolio}
 market_share_targets_portfolio %>%
+  # different targets can be plotted by setting these filters
```

Comment:
Inline comments are hard to work with. Re-styling the code can make the comment become irrelevant to the line it comments.

For example:


One style

```r
head(mtcars)  # mtcars comes with base R
```

Another style

```r
mtcars %>%
  head()   # mtcars comes with base R
```

--

I avoid this entirely by almost always avoiding inline comments and instead using top comments like this:

Before

```r
# mtcars comes with base R
head(mtcars)
```

After

```r
# mtcars comes with base R
mtcars %>%
  head()
```

I'm pretty sure there is something about this in the tidyverse style guide.

Reply:
It's expressed in a terse way but the precise interpretation leads to the same conclusion.

https://style.tidyverse.org/syntax.html

![image](https://user-images.githubusercontent.com/5856545/86247178-b6c0ab00-bb71-11ea-85d0-924ae0e36a79.png)